### ISSUE TITLE
Revisiting imbalance between 1.2.4 Captions (Live) (AA) and 1.2.9 Audio-only (Live) (AAA)

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag3#189]
This picks up something I already noted about two years ago, but could maybe be discussed in the context of WCAG 2.2/silver ... https://lists.w3.org/Archives/Public/w3c-wai-gl/2017JulSep/0052.html

In short, there's currently a weird imbalance in requiring live audio+video to have captions at AA (SC 1.2.4), while for an audio-only live stream the requirement is only at AAA. The rationale at the time, it seems, was that live audio only was more common (internet radio style) back when WCAG 2.0 was finalised, compared to the relatively more uncommon audio+video live streaming (which at the time was technically onerous, not frequently used, etc). nowadays, the opposite is likely true...there's much more audio+video live content being streamed (think twitch, let's plays, live webinars, and so on) compared to audio only. and technically, it's technically just as onerous to provide live captions for either audio only or audio+video live content.

Taking it to extreme, say an author wants to meet AA and wants to do a live vlog or something. If they just did live audio only, they'd be off the hook with regards to captions. As soon as they turn on their camera as well to show their face while vlogging, all of a sudden they're failing the live captions requirement. This seems...odd.
